front/app/api/community_monitor/keys.ts

```diff
@@ -0,0 +1,11 @@
+import { QueryKeys } from 'utils/cl-react-query/types';
+
+const baseKey = {
+  type: 'community_monitor',
+};
+
+const communityMonitorKeys = {
+  all: () => [baseKey],
+} satisfies QueryKeys;
+
```

Comment:
There's only a single community monitor on a platform, so I think we can save this just using the base key.